app/crud/chia.py

```diff
@@ -46,7 +46,7 @@ def _get_paginated_data(self, path: str, search_params: Dict[str, Any]) -> List[
         Returns:
             A list of all data retrieved from the paginated API.
         """
-        all_data = []
+        all_data: list[Any] = []
```

Comment:
list[dict[str, Any]] it seems like might be right?

Reply:
Yea, changed